Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM osrf/ros:humble-desktop-full
+
+# Install dependencies (taken from devcontainer dockerfile)
+RUN DEBIAN_FRONTEND=noninteractive apt-get update -q && \
+    apt-get update -q && \
+    apt-get install -yq --no-install-recommends \
+    lcov \
+    curl \
+    wget \
+    python3-pip \
+    python-is-python3 \
+    python3-argcomplete \
+    python3-colcon-common-extensions \
+    python3-colcon-mixin \
+    python3-rosdep \
+    libpython3-dev \
+    python3-vcstool \
```

Comment:
@ryan-roche I think we don't need no `vcstool`, dependencies are submodules.

Reply:
I will remove the install and verify that it still works

Reply:
Implemented f2e5779